### PR TITLE
Multi LVM snapshots and clones

### PR DESCRIPTION
**Pull request by shyRozen**
Multi snapshot test. Create 5 PVC, PODS, snapshots, restore PVC and attach to pod.
Need to merge [#6129](https://github.com/red-hat-storage/ocs-ci/pull/6129) to be able to execute PR validation

Signed-off-by: Shay Rozen <shay.rozen@gmail.com>

### CODE REVIEW

File: ocs_ci/helpers/helpers.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def create_pod(
     node_selector=None,
     command=None,
     command_args=None,
-    deploy_pod_status=constants.STATUS_COMPLETED,
+    deploy_pod_status=constants.STATUS_RUNNING,
```
[the remark this answers is not in the document]
Comment (shyRozen) [reply ocs_ci/helpers/helpers.py:140]: I accept because now we can set it from pod_factory. It wasn't there till now. But it is interesting who uses this more pods or dc?

Comment (Avilir) [inline ocs_ci/helpers/helpers.py:140]: why did you change this status ?
does this change impact other test ?
if you need in your test, you can pass it as parameter when you call to this function.

Comment (shyRozen) [reply ocs_ci/helpers/helpers.py]: You couldn't pass and I've added it to pod_factory. Anyway I'm dropping it. Thanks

File: tests/lvmo/test_lvm_multi_clone.py [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+    access_mode = constants.ACCESS_MODE_RWO
+    pvc_num = 5
+
+    @pytest.fixture()
+    def namespace(self, project_factory_class):
+        self.proj_obj = project_factory_class()
+        self.proj = self.proj_obj.namespace
+
+    @pytest.fixture()
+    def storageclass(self, volume_binding_mode, storageclass_factory_class):
```
Comment (ebenahar) [inline tests/lvmo/test_lvm_multi_clone.py:65]: should we take this fixture to conftest.py to be used by other tests?

File: tests/lvmo/test_lvm_multi_clone.py [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+                )
+            )
+        self.pvc_objs = []
+        for future in concurrent.futures.as_completed(futures):
+            self.pvc_objs.append(future.result())
+
+    @pytest.fixture()
+    def pod(self, pod_factory_class, volume_mode):
+        self.block = False
+        executor1 = concurrent.futures.ThreadPoolExecutor(max_workers=self.pvc_num)
```
Comment (ebenahar) [inline tests/lvmo/test_lvm_multi_clone.py:105]: I think better to name the var `executor` and not `executor1`

File: tests/lvmo/test_lvm_multi_clone.py [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+            self.pods_objs.append(future_pod.result())
+
+    @pytest.fixture()
+    def run_io(self, volume_mode):
+        self.fs = "fs"
+        self.block = False
+        if volume_mode == constants.VOLUME_MODE_BLOCK:
+            self.fs = "block"
+            self.block = True
+        executor2 = concurrent.futures.ThreadPoolExecutor(max_workers=self.pvc_num)
```
Comment (ebenahar) [inline tests/lvmo/test_lvm_multi_clone.py:124]: `executor1` is a `pod` method var and does not live outside of it so not sure we should start counting from 2 here

**Review by ebenahar — COMMENTED** [review]
Amazing work @shyRozen

File: tests/lvmo/test_lvm_multi_clone.py (hunk 4 above)
Comment (shyRozen) [reply tests/lvmo/test_lvm_multi_clone.py]: done

**Comment by shyRozen** [comment]
3 tests passed. 1 didn't run because didn't have acceptance mark. Added and reruning.

**Review by Avilir — APPROVED** [review]
LGTM

**Review by ebenahar — APPROVED** [review]
